lib/repl.js

```diff
@@ -778,6 +779,19 @@ function filteredOwnPropertyNames(obj) {
   return Object.getOwnPropertyNames(obj).filter(intFilter);
 }
 
+function getGlobalLexicalScopeNames() {
+  inspector.open();
```

Comment:
`inspector.open()` opens the WebSocket server, and is not needed for `inspector.Session`.

Reply:
Thanks, I didn't know you could use the inspector without the server xD